frontend/src/scenes/feature-flags/featureFlagLogic.tsx

```diff
@@ -200,6 +211,21 @@ export const featureFlagLogic = kea<featureFlagLogicType>({
                         },
                     }
                 },
+                setAggregationGroupTypeIndex: (state, { value }) => {
+                    if (!state || state.filters.aggregation_group_type_index == value) {
+                        return state
+                    }
+
+                    return {
+                        ...state,
+                        filters: {
+                            ...state.filters,
+                            aggregation_group_type_index: value,
+                            // :TRICKY: We reset property filters after changing what you're aggregating by.
+                            groups: state.filters.groups.map((group) => ({ ...group, properties: [] })),
```

Comment:
non-blocking

Does it make sense just to reset all the groups in this case? Seeing that I presume the groups will no longer be valid for the specific type index any more?

Reply:
Good point, will update. This will feel janky for the users and we might need to return to this at some point but /shrug